_pages/general/schools-daycare.md

```diff
@@ -1,10 +1,24 @@
 ---
-title: Schools & Daycare
-description: Details about placing children in schools and daycare
+title: Schools, Daycare and Childcare
+description: Details about placing children in schools, daycare and childcare
 ---
 
 # Schools & Daycare
 
 If you're relocating with children, you'll want to connect with someone who's recently settled their kids into a local school & daycare to help your decision-making process.
 
+# Child cares
+
+Child cares have really high rates, that's why most people tend to enroll their kids for a couple of days weekly rather than full time. In addition, due to the very high demand you may have to book your kid in a waiting list. It differs from one suburb to another, but that's pretty much how it works. It gets more trickier if both parents work full time, as they have fewer choices on the child cares with a full time availability.
+
+### Enrolling your kids
+1. To enroll your kids, first look for the child care centres around your area [Child care finder](https://www.childcarefinder.gov.au/)
+2. Fill out the applications and forms they need, and book them into the waiting lists.
+3. Once you get the place and your kid is enrolled, you will need to get a child care subsidy approved (see the next section). This only applies if you have a PR visa.
+
+### Child care subsidy
+The [Child Care Subsidy](https://www.humanservices.gov.au/individuals/services/centrelink/child-care-subsidy) is the main way the Government assists families with their child care fees.
+
+To get your child care subsidy, you will need to create a [CRN number](https://www.humanservices.gov.au/individuals/subjects/centrelink-customer-reference-number-crn) if you dont have one for yourself, spouse, and your kid. You may already have a CRN without realising. If you have used any Centrelink service in the past, it’s likely that you’ll already have one. You’ll be assigned a CRN as soon as a record is generated with Centrelink. You can find it on any of the letters sent to you from Centrelink. If you don’t have a CRN yet, you’ll need to visit your nearest service centre in order to obtain yours. Be sure to bring your [identity documents](https://www.humanservices.gov.au/customer/enablers/confirmation-identity) with you when you go. You can start applying for your CRN numbers even before enrolling into a child care.
```

Comment:
can we either use the oxford comma or not throughout the page? We haven't been consistent?

Reply:
Also, could you please update external links so they open in  new tab?